src/modules/likeApi.js

```diff
@@ -0,0 +1,38 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const appId = 'g5tsgrRdaxLZ6ykMVZ2Z';
+
+// Function to create a new like for an item
+export async function createLike(itemId) {
+  const url = `${baseUrl}apps/${appId}/likes`;
+  const data = {
```

Comment:
- [ ] Great job on successfully implementing the project requirements! 👏 Your codebase is clean and well-organized, which is commendable. However, as a suggestion for improvement, I recommend using modern JavaScript `ES6` syntax throughout your codebase.

    - One way to achieve this is by utilizing arrow functions (`=>`) instead of the traditional `function` keyword when declaring your functions. Arrow functions offer a more concise and expressive way of defining functions, which can enhance the readability of your code.

     - By adopting `ES6` syntax, your code will be more in line with current industry standards, making it easier for other developers to understand and maintain your codebase.

Overall, your work is impressive. Happy coding! 😊🚀